designs/developer/add-45/Pipelines-update/Pipelines-update.md

```diff
@@ -0,0 +1,53 @@
+---
+parent: Developer
+version: 4.5
+---
+# Add page- Updates
+
+Updates to the Add page include switching the order of the resources and pipelines section so resources come before pipelines.
+One more subtle update to the Add page is that the ‘Show pipeline visualization’ will stay open even if the user selects a new builder image or resource during their session.
+
+## Upload from Git- Updates
+
+![Add updates 1](insert image of the updated add page)
+
+A builder image is selected. When a builder image has not been selected yet, the ‘Pipelines’ section will display an info alert that says “Select a builder image and resource to see if there is a pipeline template available for this runtime.”
+
+**Note:** ‘Resources’ has been moved to the section before ‘Pipelines’.
+
+![Add updates 2](insert image of the updated add page)
+
+A builder image has been selected and the pipelines section of the Add page reacts and displays a checkbox to add a pipeline. The checkbox can be selected if the user wants to add a pipeline to their project.
+
+![Add updates 3](insert image of the updated add page)
+
+To show the pipeline visualization, the caret or label can be clicked to expand it. 
+
+![Add updates 4](insert image of the updated add page)
+
+To view more information about each of the individual pipeline sections, the user can hover over each section to get more details about it. The pipeline visualization label changes to ‘Hide pipeline visualization’ if the pipeline visualization is expanded.
+
```

Comment:
Do you mean if the user rolls over the pipeline visualization flow items (the pill shaped items)? We should ask @serenamarie125 what we call them. I know we use "sections" to refer the sections of the form.

Reply:
I can include a further mockup here to show that interaction more clearly.

Reply:
I think it is a terminology concern I have. Not that you need more documentation. We use the term "sections" to mean the sections of the form. We should just ask @serenamarie125 how to refer to the pipeline visualization components.